### PR TITLE
Add docs for forwards.

### PR DESCRIPTION
**Pull request by judepereira**

### CODE REVIEW

File: schema/expose/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Expose
+
+To indicate to a cloud provider that certain endpoints of your micrservice
+may be exposed to the public internet, an `expose` MAY be declared.
```
Comment (wilzbach) [inline schema/expose/README.md]: An expose _what_? A block, section, ...

File: schema/expose/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Any HTTP traffic coming to this endpoint from an external source (the internet)
+SHOULD be directly proxied to your container.
+
+An `expose` may be defined as the following:
+```yaml
+omg: 1
+lifecycle: ...
+expose:
```
Comment (wilzbach) [inline schema/expose/README.md]: Shouldn't this be plural `exposes` like the existing actions, commands, arguments and events?

File: schema/expose/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<p>
+{
+    "path": {
+        "required": true, 
+        "desc": "The HTTP path for this `expose`"
+    },
+    "port": {
+        "required": true, 
+        "desc": "The port for this `expose`"
+    }
```
Comment (wilzbach) [inline schema/expose/README.md]: No help?

Comment (judepereira) [reply schema/expose/README.md]: Good point, I need to add this.

**Comment by kenneth-reitz** [comment]
what's the status of this?

**Comment by judepereira** [comment]
@kenneth-reitz just need to rename `expose` to `forward` everywhere, and merge it.

**Comment by kenneth-reitz** [comment]
Awesome! I'll give it a try soon :)

File: schema/expose/README.md (hunk 2 above)
Comment (judepereira) [reply schema/expose/README.md]: Renamed to `forwards` :)

**Pull request merged**